packages/core/src/types.ts

```diff
@@ -230,19 +230,44 @@ export type RecursiveDesignTokenDefinition = {
   [key: string]: string | RecursiveDesignTokenDefinition;
 };
 
+/** not yet implemented, don't use */
```

Comment:
I think the `DesignBorderTokenStyle` is good to go as nothing changed, it's just reassigning the values to a type